tests/providers/google/cloud/operators/test_functions.py

```diff
@@ -711,4 +711,9 @@ def test_execute(self, mock_gcf_hook, mock_xcom):
             function_id=function_id, input_data=payload, location=GCP_LOCATION, project_id=GCP_PROJECT_ID
         )
 
-        mock_xcom.assert_called_once_with(context=None, key='execution_id', value=exec_id)
+        mock_xcom.assert_has_calls(
+            [
+                mock.call(context=None, key='execution_id', value=function_call_response['executionId']),
+                mock.call(context=None, key='function_response', value=function_call_response['result'])
```

Comment:
Testing looks good, but there are static checks that are failing. Specifically a missing comma here.

Enable pre-commit tests (see [docs here](https://github.com/apache/airflow/blob/main/STATIC_CODE_CHECKS.rst#installing-pre-commit-hooks)) so that you can catch these errors locally on your dev machine.